packages/ripple-binary-codec/src/enums/README.md

```diff
@@ -1,22 +1,24 @@
 # Definitions
 
-## Types
+## Overview of Terms
+
+### Types
```

Comment:
Why is this change made? It doesn't look like there are any other subheadings under `Overview of Terms`

Reply:
I forget :P - Doesn't seem necessary.